calculator/src/main/java/edu/nextstep/camp/calculator/CalculatorViewModel.kt

```diff
@@ -0,0 +1,44 @@
+package edu.nextstep.camp.calculator
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import edu.nextstep.camp.domain.Calculator
+import edu.nextstep.camp.domain.Expression
+import edu.nextstep.camp.domain.Operator
+
+class CalculatorViewModel : ViewModel() {
+    private val calculator = Calculator()
+
+    private val _expressionEvent = SingleLiveEvent<Expression>()
+    val expressionEvent: LiveData<Expression> get() = _expressionEvent
+
+    private val _errorEvent = SingleLiveEvent<Void>()
+    val errorEvent: LiveData<Void> get() = _errorEvent
```

Comment:
```suggestion
    private val _errorEvent = SingleLiveEvent<Unit>()
    val errorEvent: LiveData<Unit> get() = _errorEvent
```
코틀린에서는 void를 사용하지 않고 Unit을 사용합니다.

Reply:
지금은 에러가 하나이지만 여러 종류의 에러를 다루게 된다면 어떻게 처리하면 좋을까요?

Reply:
말씀하신대로 에러가 여러 개인 경우에는 sealed class를 활용하면 될 것 같아요. 👍 
알고 계신 사항이라면 수정을 하지 않으셔도 될 것 같아요 😄